Legion/ArtifactScenarios/Locales/zhCN.lua

```diff
@@ -64,7 +64,7 @@ if L then
 	L.name = "大法师克希雷姆"
 	L.corruptingShadows = "腐蚀暗影"
 
-	--L.warmup_trigger1 = "You are too late, warrior! With the Focusing Iris under my control, I can siphon the arcane energy from Azeroth's ley lines directly into my magnificent self!"
+	L.warmup_trigger1 = "你太迟了，<class>！掌握了聚焦之虹，我就能直接从艾泽拉斯的魔网中抽取奥术能量来强化自身的法力！"  --猎人进本NPC喊的是猎人，<class>是任务插件Questie任务描述常用的。
```

Comment:
This is a trigger so must match exactly what the NPC says in game, this probably never worked for classes besides warrior.

These classes are eligible for this scenario:
Frost Death Knight
Havoc Demon Hunter
Survival Hunter
Subtlety Rogue
Arms Warrior

It seems like this will need a logic update to work properly for all classes.